skyportal/models/comment.py

```diff
@@ -13,6 +13,22 @@
 from .group import accessible_by_groups_members
 
 
+"""
+NOTE ON ADDING NEW COMMENT TYPES:
+To add a new comment on <something> you need to
+- Inherit from CommentMixin (as well as Base).
+- Add the name of the table to backref_name on CommentMixin
+- Add any additional columns, like references to a model the comment is on.
+- Add the comment as a relationship with back_populates (etc.) on the model you are commenting on.
+  (e.g., for CommentOnSpectrum you need to add "comments" to models/spectrum.py)
+- Add a join to models/group_joins.py so comments will have groups associated with them.
+- Add a join to models/user_token.py so comments will have a user associated with them.
+- Update the API endpoints for comments, and the reducers to listen for changes in the comments.
+- Update the app_server.py paths to accept the new type of comment upon API calls.
+
+"""
+
+
```

Comment:
This is good instruction for the future. But it should be moved to the development docs (`doc/adding_comment_types.md`?), not as a docstring.

Reply:
I think that having it also as a docstring would be more useful to future developers, that sometimes (only very rarely) don't read the docs but do look at the code. Maybe we can have this in both places?